test/ipsecconf_test.rb

```diff
@@ -208,7 +208,7 @@
         it "Creates a SuSE firewall script for all connections" do
             # Set reduce MSS to true
             exported = Yast::IPSecConf.Export
-            exported["tcp_mss_1024"] = true
+            exported["tcp_reduce_mss"] = true
             Yast::IPSecConf.Import(exported)
             expect(Yast::IPSecConf.GenFirewallScript).to eq("""# The file is automatically generated by YaST VPN module.
```

Comment:
Warning, there is no triple double quote in Ruby! You are actually concatenating empty strings to an ordinary multiline string.
```console
$ ruby-parse -e 'puts """Hello"""'
(send nil :puts
  (dstr
    (str "")
    (str "Hello")
    (str "")))
```
It is better to write this with a heredoc, where the single quotes additionally prevent interpolation
```rb
expect(...).to eq << 'EOS'
# The file is automatically 
...
true
}
fw_custom_after_finished
EOS
```